src/shared/cloudformation/yamlExtension.ts

```diff
@@ -0,0 +1,68 @@
+/*!
```

Comment:
IIUC this is a wrapper for the API of a third party extension. I think we should keep all such wrappers in a single place, e.g. `src/shared/thirdparty/*`, thus this would live in `src/shared/thirdparty/yamlExtension.ts`

Reply:
I was actually thinking `src/shared/extensions` though `thirdparty` makes sense too.

Reply:
either is fine